plasmapy/examples/plot_physics.py

```diff
@@ -57,4 +57,79 @@
     formulary.plasma_frequency(electron_concentration))
 plt.xlabel("Electron Concentration (m^-3)")
 plt.ylabel("Langmuir Wave Plasma Frequency (rad/s)")
+plt.grid()
```

Comment:
Come to think of it, it would probably make sense to plot this on a log-log scale, since our data spans multiple orders of magnitude:
```suggestion
plt.grid()
plt.xscale('log')
plt.yscale('log')
```